scripts/microbe_stage/organelle.as

```diff
@@ -733,7 +740,7 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
             // Create the matrix with the offset
             Ogre::Matrix4 hexFinalOffset(translation);
```

Comment:
This needs to be affected by also if this is a bacteria or not.

Reply:
Is this the offset outside the membrane?

Reply:
The matrix is for the graphical view. The `AddChildShape` uses `translation` to mark the relative offset of the child physics body from the origin of the parent body.